src/Core/src/Platform/iOS/AcceleratorExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Linq;
+using Foundation;
+using ObjCRuntime;
+using UIKit;
+
+namespace Microsoft.Maui.Platform
+{
+	internal static class AcceleratorExtensions
+	{
+		internal static UIMenuElement GetPlatformAccelerator(this IMenuFlyoutItem virtualView, IMauiContext mauiContext)
+		{
+			int index = MenuFlyoutItemHandler.menus.Count;
+			UIImage? uiImage = virtualView.Source.GetPlatformMenuImage(mauiContext);
+			var selector = new Selector($"MenuItem{index}:");
+
+			bool selectorFound =
+				MauiUIApplicationDelegate.Current.RespondsToSelector(selector);
+
+			if (!selectorFound)
+			{
+				throw new InvalidOperationException(
+					$"By default we only support 50 MenuItems. You can add more by adding the following code to {MauiUIApplicationDelegate.Current.GetType()}\n\n" +
+					$"[Export(\"MenuItem{index}: \")]\n" +
+					$"internal void MenuItem{index}(UICommand uICommand)\n" +
+					"{\n" +
+					"	uICommand.SendClicked();\n" +
+					"}");
+			}
+
+			var accelerator = virtualView.Accelerator;
+
+			if (accelerator is not null)
```

Comment:
return right away the last line instead of another nesting 

```suggestion
			if (accelerator is null)
			    return virtualView.CreateMenuItemCommand(index, uiImage, selector);
```

Reply:
nice, thanks